README.md

```diff
@@ -1823,7 +1823,7 @@ Sets the list of resources to look for when a client requests an index of the di
 
 ##### `docroot`
 
-**Required**. Sets the [`DocumentRoot`][] location, from which Apache serves files.
```

Comment:
i'm still unclear in how far this is no longer **Required**, since in vhost.pp, we still have this:

```puppet
 define apache::vhost(
   $docroot,
   $manage_docroot              = true,
  …
```